style/amenity-points.mss

```diff
@@ -1383,6 +1383,12 @@
     marker-clip: false;
   }
 
+  [feature = 'mountain_pass'][zoom >= 15] {
+    marker-file: url('symbols/natural/saddle.svg');
+    marker-fill: @transportation-text;
```

Comment:
`transportation-icon` is the color for symbols, `transportation-text` is the color for the corresponding labels:

https://github.com/gravitystorm/openstreetmap-carto/blob/faead203a2fb4400050d59ac2d5ded7ff5bcd003/style/amenity-points.mss#L5-L6